web/src/Web.App/Views/Shared/_RagRatingCommentary.cshtml

```diff
@@ -0,0 +1,15 @@
+@using Web.App.Domain
+@using Web.App.Extensions
+@model Web.App.ViewModels.RagRatingCommentaryViewModel
+
+Spending is
+@if (Model.Rating.RAG == "red" && Category.InvertRagValueCategories.Contains(Model.Rating.Category))
```

Comment:
Incorrect logic: here: the message changes based on the percentile not the RAG.

```
Above or equal 50%
Spends £[amount] per [unit] — Spending is higher than [precent]% of similar schools.
Below 50%
Spends £[amount] per [unit] — Spending is less than [100-precent]% of similar schools.

Above or equal 50%
Spending is higher than [percent]% of similar schools.
Below 50%
Spending is less than [100-percent]% of similar schools.
```

Reply:
Good spot. Thanks for clarifying on the story, also